### ISSUE TITLE
Possible support for Legacy Console world generation (Xbox 360 TU31/PS4 1.95 Village and Pillage)

### ISSUE THREAD

**Issue by Delvin4519** [BlueStaggo/moderner-beta#3]
In Legacy Console Edition biomes generate 4 times smaller, but it does not affect rivers or hill biomes.

In Java 1.17.1, biomes are scaled twice between 1:256 Biome and 1:64 Hills. In Legacy Console Edition, this scaling does not occur with using small biomes, and only once with "LCE medium biomes". Biome scaling and Hill scaling in the moderner beta mod affects rivers when rivers should be using a separate scaling.

Note: Prior to 1.13, in Java 1.12.2 and below, M biomes generate using 0 for the "layer salt seed" as well as using 0 for it's "world seed". This bug was fixed in Java 1.13. Replicating Legacy Console Edition M biomes requires that this "bug fix" be undone and reverted, as M biomes on LCE generate using 0 for the layer salt seed and 0 for its "world seed". 

Using MCP-Reborn on Java 1.17.1 was quite trivial in adding the 0 seed bug for M biomes and downscaling biomes to match LCE.

Java 1.17.1 modded to match LCE biomes (left), LCE biomes (middle), moderner-beta mod set to biome size 2 (right)
![image](https://github.com/b3spectacled/modern-beta-fabric/assets/23464599/5b9f82a2-228b-41c8-9e95-743e1b17ef1f)

On the left is a test case of how LCE biomes should generate (pay attention to the river, hills, and M biomes) using seed 306959825. On LCE they have the aquatic update and so aquatic biomes should be included in LCE.
![image](https://github.com/b3spectacled/modern-beta-fabric/assets/23464599/951d7c81-4fc8-4ede-8da8-5ddaeb039893)

**Comment by DarianLStephens** [comment]
Speaking of legacy console, it appears that Moderner Beta didn't carry over the last updates of Modern Beta, which added the original console edition worlds, the ones with limited size.
I quite enjoy having a smaller world with defined limits, and losing that is kinda sad.

**Comment by forkiesassds** [comment]
I'm curious on where is the bug fixed, as searching through 1.9 code I can't seem to find any 0 seed bug anywhere.

**Comment by forkiesassds** [comment]
> Speaking of legacy console, it appears that Moderner Beta didn't carry over the last updates of Modern Beta, which added the original console edition worlds, the ones with limited size. I quite enjoy having a smaller world with defined limits, and losing that is kinda sad.

This is not true, the preset is in the mod as it is based on the last commit of the original mod.

**Comment by DarianLStephens** [comment]
Where is it located? I haven't been able to find the option.

**Comment by forkiesassds** [comment]
> Where is it located? I haven't been able to find the option.

In the Beta Custom category.
